RecoLocalMuon/Configuration/python/RecoLocalMuon_cff.py

```diff
@@ -37,11 +37,15 @@
 # DT, CSC and RPC together (correct sequence for the standard path)
 muonlocalreco = cms.Sequence(dtlocalreco+csclocalreco+rpcRecHits)
 
-def _modifyRecoLocalMuonForPhase2( theProcess ):
-    theProcess.load("RecoLocalMuon.GEMRecHit.gemRecHits_cfi")
-    theProcess.load("RecoLocalMuon.GEMRecHit.me0LocalReco_cff")
-    theProcess.muonlocalreco += theProcess.gemRecHits
-    theProcess.muonlocalreco += theProcess.me0LocalReco
+from RecoLocalMuon.GEMRecHit.gemRecHits_cfi import gemRecHits
+from RecoLocalMuon.GEMRecHit.me0LocalReco_cff import me0LocalReco
+
+_run3_muonlocalreco = muonlocalreco.copy()
+_run3_muonlocalreco += gemRecHits
+
+_phase2_muonlocalreco = _run3_muonlocalreco.copy()
+_phase2_muonlocalreco += me0LocalReco
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesRecoLocalMuonForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyRecoLocalMuonForPhase2 )
+eras.run3_GEM.toReplaceWith( muonlocalreco , _run3_muonlocalreco )
+eras.phase2_muon.toReplaceWith( muonlocalreco , _phase2_muonlocalreco )
```

Comment:
@Dr15Jones could it be this line is interfering with 

<PRE>
from RecoMuon.MuonIdentification.me0MuonReco_cff import me0MuonReco
_phase2_muonGlobalReco = muonGlobalReco.copy()
_phase2_muonGlobalReco += me0MuonReco
eras.phase2_muon.toReplaceWith( muonGlobalReco, _phase2_muonGlobalReco )
</PRE>

If so, what is the recommended way to fix it?